modules/nlu/src/backend/pipelines/entities/pattern_extractor.ts

```diff
@@ -38,37 +40,44 @@ export default class PatternExtractor {
   ): Promise<sdk.NLU.Entity[]> {
     const values = await Promise.all(
       [occurence.name, ...occurence.synonyms].map(async x =>
-        (await this.languageProvider.tokenize(x.toLowerCase(), ds.language)).map(sanitize).filter(t => t.length)
+        (await this.languageProvider.tokenize(x.toLowerCase(), ds.language)).filter(t => t.length)
```

Comment:
Probably just a personal preference but I prefer .filter(_.isEmpty) instead of t.length

Reply:
Both are opposite. I actually want to keep tokens which are not empty.